### PR TITLE
🍒 Fix scrollbar interaction in the custom expression suggestion list (#43837)

### PR DESCRIPTION
**Pull request by nemanjaglumac**
Manual backport of the external fix.
https://github.com/metabase/metabase/pull/43837

Thanks once again @hyperNURb

### CODE REVIEW

**Review by deniskaber — APPROVED** [review]

**Review by kamilmielnik — CHANGES_REQUESTED** [review]
Please don't merge it, see #43992

**Comment by nemanjaglumac** [comment]
> Please don't merge it, see #43992

We can't merge even if we wanted to because of the failing tests.
Once the follow up fix is merged, I'll cherry pick it here.

**Pull request closed without merging**
